include-what-you-use.1

```diff
@@ -213,13 +215,13 @@ Includes the contents of another
 .PP
 The descriptions of headers and symbols are as follows:
 .TP
-.IB "header\fR := " \(dq include-spec "\(dq, " \(dqvisibility\(dq
+.IB "header\fR := " \(dq include-spec "\(dq, \(dq" visibility \(dq
 Describes a header file. The
 .I include-spec
 specifies the header file and
 .I visibility
 specifies whether the header is
-.BR \(dqpublic\(dq " or " \(dqprivate\(dq .
+.BR public " or " private .
```

Comment:
Could you do similar quoting on line 241 where symbol mappings are described to have *private* by default?

Reply:
Sorry, I had my diffs backward. It's consistent after your change, so no change needed.